index.bs

```diff
@@ -6438,6 +6445,90 @@ requests will be affected.
 
 </div>
 
+#### The network.setCacheMode Command ####  {#command-network-setCacheMode}
+
+The <dfn export for=commands>network.setCacheMode</dfn> command overrides
+the network cache behaviour for certain requests.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      network.SetCacheMode = (
+        method: "network.setCacheMode",
+        params: network.SetCacheModeParameters
+      )
+
+      network.SetCacheModeParameters = {
+        mode: "no-store" / null,
+        ? contexts: [+browsingContext.BrowsingContext]
+      }
+    </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl">
+      EmptyResult
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm>
+The <dfn export>WebDriver BiDi update request cache mode</dfn> steps given
+|request| are:
+
+1. Let |context| be null.
+
+1. If |request|'s [=request/window=] is an [=environment settings object=]:
+
+  1. Let |environment settings| be |request|'s [=request/window=]
+
+  1. If there is a [=/browsing context=] whose [=active window=] is |environment
+     settings|' [=environment settings object/global object=], set |context| to
+     the [=top-level browsing context=] for that context.
+
+1. If |context| is not null:
+
+  1. If [=cache mode override map=] [=map/contains=] |context|, set
+     |request|'s [=request/cache mode=] to [=cache mode override
```

Comment:
so one difference to the CDP implementation and what Puppeteer relied on is that in CDP this command also disabled the memory cache which does not seem to be specified https://github.com/whatwg/fetch/issues/1400 there are some definitions for the memory image cache https://html.spec.whatwg.org/multipage/images.html#the-list-of-available-images so perhaps we should include it at least?